app/services/typicode.js

```diff
@@ -0,0 +1,27 @@
+const request = require('request-promise'),
+  typicodePath = 'https://jsonplaceholder.typicode.com',
+  apiError = require('../errors');
+
+function options(endpoint) {
```

Comment:
Try to stick to one type of function for the entire project. Either use arrow functions (except for model prototypes) or use classic functions.